core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/RawlsWorkspaceAclManager.scala

```diff
@@ -119,4 +123,30 @@ class RawlsWorkspaceAclManager(val samDAO: SamDAO)(implicit val executionContext
                            ctx: RawlsRequestContext
   ): Future[Unit] =
     samDAO.removeUserFromPolicy(SamResourceTypeNames.workspace, workspace.workspaceId, policyName, email.value, ctx)
+
+  def maybeShareWorkspaceNamespaceCompute(policyAdditions: Set[(SamResourcePolicyName, String)],
+                                          workspaceName: WorkspaceName,
+                                          ctx: RawlsRequestContext
+  ): Future[Unit] = {
+    val newWriterEmails = policyAdditions.collect { case (SamWorkspacePolicyNames.canCompute, email) =>
```

Comment:
This could also be converted to the syntax in `MultiCloudWorkspaceACLManager`, if you feel so inclined.